### DISCUSSION TITLE
Output data

### DISCUSSION THREAD

**Discussion by SifonFeramon** [colmap/glomap#24] [General]
How to read output data, such as cameras.bin, points3D.bin?

**Comment by ahojnnes** [comment]
The output is written in colmap format. Please refer to the colmap documentation for how to parse and interpret the data.

**Comment by SifonFeramon** [comment]
thank you, i check it.

**Comment by eyildiz-ugoe** [comment]
you need to use model converter to obtain a .ply file by passing the /sparse folder, which is your output from glomap. I use something like this:

`run(colmap, f"model_converter --input_path '{work_dir}/sparse/0' --output_path '{work_dir}/sparse/0/sparse.ply' --output_type PLY")`
